### DISCUSSION TITLE
coroutine support for awaitable operations

### DISCUSSION THREAD

**Discussion by mike-5345** [fnc12/sqlite_orm#1330] [Ideas]
just curious if it would be possible to support coroutines for awaitable operations

**Comment by fnc12** [comment]
it depends whether it is possible to support coroutines with raw SQLite

**Reply by mike-5345** [reply]
Thanks for getting back to me. Just another quick question: is it possible in the lib to insert/update if the record does not or does exist?

**Reply by fnc12** [reply]
the same: it depends on how you see it with raw SQLite. Please tell me the SQL query you want to run and I'll tell you whether we support it

**Comment by trueqbit** [comment]
> just curious if it would be possible to support coroutines for awaitable operations

@mike-5345 What is your requirement?
